### PR TITLE
Feature speaker doc

### PR DESCRIPTION
**Pull request by DavidMansolino**
New Speaker node documentation.
Linked with omichel/webots#3677

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

**Comment by omichel** [comment]
:+1:

**Comment by omichel** [comment]
The node chart should also be updated to add the Speaker node: https://www.cyberbotics.com/doc/reference/chart?version=feature-speaker-doc

**Comment by DavidMansolino** [comment]
Yes indeed, I will update it.

File: reference/speaker.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+Derived from [Device](device.md).
+
+```
+Speaker {
+}
+```
+
+### Description
+
+The [Speaker](#speaker) node represents a loudspeaker device that can be embbed onboard a robot or standing in the environment. It can be used to play sounds and perform text-to-speech from the controller API.
```
Comment (DavidMansolino) [inline reference/speaker.md]: typo: embbed

File: reference/speaker.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+It is possible to change the volume, pitch, balance, and loop parameters of a sound currently playing by calling again the `wb_speaker_play_sound` function with the same speakers and `sound` arguments.
+
+> **note**:
+The path to the sound file should be defined either absolutely or relatively. If defined relatively, it be searched first relatively to the robot controller folder. If not found there and if the robot is a PROTO, it will be searched relatively to the PROTO folder of the robot.
```
Comment (DavidMansolino) [inline reference/speaker.md]: it be searched: it seems a word is missing

**Comment by DavidMansolino** [comment]
:+1:

**Comment by omichel** [comment]
:+1:

**Pull request closed without merging**